allow changing camera orientation and crop

nohn opened the pull request.

nohn committed:
allow changing camera orientation

nohn committed:
remove debugging output

Anonymousdog commented:
Cool mod!  I'd want a global boolean variable to switch on/off support for orientation values and to include logic for setting safe, default values if the layout variables are unset (before merging these changes).

nohn committed:
add camera crops functionality

nohn committed:
check if variable is set, otherwise ignore

nohn commented:
Done

nohn commented:
Also added support for cropping.

bal0u45 committed:
Change formatting

bal0u45 committed:
Fix missing variable

Adding missing $camera_crop variable to the repair part and changing the $camera_orientation variable to work like in the startup part

bal0u45 committed:
Revert "Change formatting"

This reverts commit 3692d162935f4c1130eac50356bba7a0a424cde3.

nohn committed:
fix repair mode

nohn committed:
Merge pull request #1 from bal0u45/master

Fix missing variable

coltenkrauter commented:
@Anonymousdog thoughts on merging this pr?

nohn committed:
Merge branch 'Anonymousdog:master' into master
